fix_admin_credits_simple.py

```
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""
+Simple script to check and fix admin credits
+"""
+
+import requests
+import json
+
+# Configuration
+BASE_URL = "https://cryptouniverse.onrender.com/api/v1"
+ADMIN_EMAIL = "admin@cryptouniverse.com"
+ADMIN_PASSWORD = "AdminPass123!"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "ADMIN_PASSWORD"

(S105)